libs/utils/src/decorators/is-schema-id.decorator.ts

```diff
@@ -0,0 +1,26 @@
+import { registerDecorator, ValidationArguments, ValidationOptions } from 'class-validator';
+
+export function IsSchemaId(validationOptions?: ValidationOptions) {
+  // eslint-disable-next-line func-names
+  return function (object: object, propertyName: string) {
+    registerDecorator({
+      name: 'IsSchemaId',
+      target: object.constructor,
+      propertyName,
+      options: validationOptions,
+      validator: {
+        validate(value: unknown, _args: ValidationArguments) {
+          const re = /^[1-9][0-9]*$/;
```

Comment:
What's our position on leading zeroes? This precludes them, but maybe they should be OK?